README.md

```diff
@@ -1,50 +1,41 @@
 # HTML5 Semantic Elements
 
-## Overview
+## Problem Statement
 
-In this lesson we will learn about semantic elements, their purpose, and best practices for using them.
+We have `div` elements to organize and label sections of HTML. But have you wondered if having so many `div` tags makes our HTML document look cluttered and confusing? Even when we identify or classify them with specific names, it's a lot to keep track of. It would be clearer for both developers and browsers if we could use more descriptive HTML elements to arrange our content. That's where semantic elements come in.
 
 ## Objectives
 
-1. Historical significance and why semantic elements were created.
-2. Familiarization with HTML5 Semantic Elements.
+1. Explain the historical reasoning behind semantic elements
+2. Identify specific HTML5 semantic elements
  
-## A Better Way To Markup Content
+## Explain the Historical Reasoning Behind Semantic Elements
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/V28xLFEYaSQ?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe>
-
-*Note: Slides for this lecture video are provided in the resources at the bottom of this lesson.*
-
-### A Brief History
-
-At the dawn of HTML, developers had but one common element that was used to create generic boxes and that my friend was the `<div>` element. Pages layouts at that time then became a sea of divs. This worked, but we needed a way to label them to distinguish one div from another. This was largely done with id and class names such as 
+When developers first began defining containers to structure HTML, they had only one generic element available to them: the `div`. Creating complex page layouts then required dozens of `div` elements that were often difficult to organize or locate within the code. They needed a way to distinguish one `div` from another, which led to the use of `id` and `class` names to specify what they intended the `div` to contain.
```

Comment:
> ...led to the use of `id` and `class` names

1. That's not _quite_ accurate, i think `names` should be reworked as "attributes e.g.

2. Idea for rework:

> which led to the use of `id` and `class` attributes on elements being, well, misused in an attempt to communicate what the `<div>` was doing. For example, a document usually has only one header so it seemed sensible to say `<div id="header">`. For a list of Tweets we might have said `<div class="tweet">. As authors we meant to say something stronger than "this is a text division" we wanted to say "this is a special bloc of introduction." Technologists would say we wanted those blocs to have a _semantic_ meaning.